java/src/jmri/jmrix/ConnectionConfig.java

```diff
@@ -48,7 +48,13 @@
      * </ul>
      */
     public void register();
-    
+
+    /**
+     * Load the adapter with an appropriate object
+     * <i>unless</I> its already been set.
+     */
```

Comment:
Is there a reason this is needed systematically?

setInstance often means setting a _static_ instance, which we’re trying to get rid of as part of multi-connection support. This seems to be a per-object, not static, thing, but then I’m not sure what “instance” means.